ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -315,6 +315,9 @@ def prod(
             out=out,
         )
 
+    def tofile(self, fid, sep="", format="%s"):
+        return self._ivy_array.to_file(fid, sep=sep, format=format)
```

Comment:
@cressidasuphina A small error here; the parameter should be format_ , not format (if you check the to_file() method in array/general.py you will see why)

Reply:
@Fayad-Alman I'm sorry, I haven't noticed.